var/spack/repos/builtin/packages/ucx/package.py

```diff
@@ -25,6 +26,17 @@ class Ucx(AutotoolsPackage):
     version('1.3.0', sha256='71e69e6d78a4950cc5a1edcbe59bf7a8f8e38d59c9f823109853927c4d442952')
     version('1.2.2', sha256='914d10fee8f970d4fb286079dd656cf8a260ec7d724d5f751b3109ed32a6da63')
     version('1.2.1', sha256='fc63760601c03ff60a2531ec3c6637e98f5b743576eb410f245839c84a0ad617')
+    version('1.2.0', sha256='1e1a62d6d0f89ce59e384b0b5b30b416b8fd8d7cedec4182a5319d0dfddf649c')
+
+    variant('thread_multiple', default=False,
+            description='Enable thread support in UCP and UCT')
 
     depends_on('numactl')
     depends_on('rdma-core')
+
+    def configure_args(self):
+        spec = self.spec
+        config_args = []
+        if '+thread_multiple' in spec:
+            config_args.append('--enable-mt')
```

Comment:
Can you explicitly disable this in an else clause?

Reply:
Done as suggested.